user_guide_src/source/database/configuration.rst

```diff
@@ -29,13 +29,13 @@ prototype:
 The name of the class property is the connection name, and can be used
 while connecting to specify a group name.
 
-.. note:: The default location of the SQLite3 database is in the **writable** folder.
-    If you want to change the location, you must set the full path to the new folder.
+.. note:: The default location of the SQLite3 database is the **writable** folder.
+    If you want to change the location, you must set the full path to the new folder (e.g., 'database' => '../db/database_name').
```

Comment:
1. `../db/database_name` is not a full path, but a relative path.
2. It is better to have the file extention.

Change to something like `WRITEPATH . 'db/database_name.db'`.

Reply:
@kenjis Changed. Thanks.